test/sampler.jl

```diff
@@ -27,3 +29,7 @@ Nested(ndims=3, nactive=100, enlarge=1.25, update_interval=150)
 spl = Nested(10, 1000)
 @test spl.proposal isa Proposals.RWalk
 @test spl.update_interval == 3750
+
+spl = Nested(10, 1000)
+@test spl.proposal isa Proposals.RStagger
+@test spl.update_interval == 3750
```

Comment:
This test asserts behavior that is not true- the sampler proposal _should not_ be `RStagger` given these inputs- it should be `RWalk`.

This ties back to my point that for the automatic proposal definition we're never going to pick `RStagger` (using our current heuristic) so there's no need to comment it/test it.

In addition, you should try running tests locally if you can- I can tell without even running this test fails, but for you this might point out a bug in the code or in the tests without needing an extra review to find it!

Reply:
yes, removing RStagger from here. also will run tests locally henceforth